docs/reference/schema.md

```diff
@@ -101,6 +101,10 @@ Specifies the list of ingresses.
 
 Specifies the list of services. Applications must have at least one service.
 
+#### `solution` (string)
+
+Indicates the solution to use when building project-based services in watch mode. If omitted, those services will be built individually. Specifying the solution can help reduce repeated builds of shared libraries when in watch mode.
```

Comment:
We use the method `SolutionFIle.Parse(solutionPath)` for processing this property. This can process both `.sln` and Solution Filter (`.slnf`). This should be indicated here in the documentation.

Reply:
Updated text.

Reply:
I think, you missed the push.

Reply:
Well, that was bizarre; VS Code insisted it was up to date.  Anyway, pushed the hard way.